src/makeRobot.js

```diff
@@ -38,7 +38,70 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
-}
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+
+    get location() {
+      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward: function(value = 1) {
+      if (value < 0) {
+        return this;
+      }
+
+      this.coords.y += value;
+
+      return this;
```

Comment:
```suggestion
      if (value > 0) {
        this.coords.y += value;
      }

      return this;
```
Just add the coords only if value is positive